corehq/apps/hqadmin/reports.py

```diff
@@ -541,7 +543,7 @@ class AdminReport(GenericTabularReport):
     @use_bootstrap3
     @use_datatables
     def set_bootstrap3_status(self, request, *args, **kwargs):
```

Comment:
and I don't want to keep it separate from the `ReportDispatchers` as it's more clear up front what's been migrated...